recipes/cpptrace/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(test_package CXX)
+
+find_package(cpptrace REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+
+target_link_libraries(${PROJECT_NAME} PRIVATE cpptrace::cpptrace)
```

Comment:
Maybe add a `target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)` to ensure it works in C++11?